### PR TITLE
Pacman

### PR DESCRIPTION
**Pull request by Valansch**

### CODE REVIEW

**Comment by TWLTriston** [comment]
Added ship wreck loot as part of this feature branch

**Comment by Valansch** [comment]
Oh good, i dont have to approve, you already did that! xD

File: map_layout.lua [hunk 1]
```diff
@@ -62,8 +64,8 @@ local on_chunk_generated = function(event)
 				run_ores_module(event)
 			end
 		end
-		for _,v in pairs(miscs) do
-			v.on_chunk_generated(event)
+		if run_misc_module ~= nil then
+			run_misc_module(event)
 		end
```
Comment (Valansch) [inline map_layout.lua]: Please revert this and implement the misc module compatible with the existing code. This code breaks other misc modules and breaks being able to use multiple misc modules. Map_layout.lua is working as inteded. Look in rail_grid.lua on how to create a misc module.

**Comment by TWLTriston** [comment]
Rebase, pulled RSO in and play tested

**Comment by Valansch** [comment]
Looks very good thank you. Merged

**Pull request merged**